test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -310,6 +320,9 @@ def _undefined
 
   # Set up grpc stubs to mock the external calls.
```

Comment:
Let's document the (hopefully temporary) constraint that a we should create exactly one driver within this context.

Reply:
Documented.